rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -263,7 +265,11 @@ private UserAuthorizationVerificationTask(Context context) {
         @Override
         protected Void doInBackground(Void... voids) {
             boolean isUserAuthorized = syncUtils.verifyAuthorization();
-            if (!isUserAuthorized) {
+
+            Activity covidJsonFormActivity = RDTApplication.getInstance().getCurrentActivity();
```

Comment:
Rename to `currentActivity` from `covidJsonFormActivity`

Reply:
fixed.